CONTRIBUTING.md

```diff
@@ -24,21 +25,28 @@ Contributions are essential for keeping this extension great. We try to keep it
               ├──── vscode-xml/
   ```
 
-3. `cd vscode-xml/`
-   
-4. Install the dependencies:
+3. `cd lsp4xml/`
+
+4. Install the maven dependencies:
+	```bash  
+	$ mvn install
```

Comment:
`./mvnw verify`
or 
`mvnw.cmd verify` on windows

You hardly ever need to call install (that will install your build to the local Maven repository, to be consumed by other projects depending on it)